symbols/Espressif.kicad_sym

```diff
@@ -4350,6 +4350,692 @@
 			)
 		)
 	)
+	(symbol "ESP32-C5-DevKitC-1"
```

Comment:
The symbol pins do not look aligned with the box. Can you double-check?

![Screenshot 2025-02-04 at 10 16 22](https://github.com/user-attachments/assets/a08e7e97-32a6-4174-a81c-25743ccb6129)